sdk/core/Azure.Core/src/Pipeline/HttpClientTransport.cs

```diff
@@ -83,13 +85,16 @@ private async ValueTask ProcessAsync(HttpMessage message, bool async)
 #if NET5_0
                 if (!async)
                 {
-                    responseMessage = _client.Send(httpRequest, HttpCompletionOption.ResponseHeadersRead, message.CancellationToken);
+                    // Sync HttpClient.Send is not supported on browser but so is the sync-over-async
+#pragma warning disable CA1416 // 'HttpClient.Send(HttpRequestMessage, HttpCompletionOption, CancellationToken)' is unsupported on 'browser'
+                    responseMessage = Client.Send(httpRequest, HttpCompletionOption.ResponseHeadersRead, message.CancellationToken);
```

Comment:
Would a `browser` client just never call anything sync to hit this path?

Reply:
Yeah, sync methods won't ever work.